lightning/src/ln/peer_handler.rs

```diff
@@ -1088,15 +1094,75 @@ impl<Descriptor: SocketDescriptor> PeerManager<Descriptor> {
 			}
 		};
 	}
+/// This function can be called every 30 seconds or so, it will:
```

Comment:
Can you clean up the indentation? There are a number of things that could be cleaned up to make the code more readable.

Reply:
no problem!